From fd6ecc72a252f62dc92ffdf28e81e34a587a1758 Mon Sep 17 00:00:00 2001
From: weidongliu <weidongliu@chromium.org>
Date: Fri, 14 Mar 2025 21:02:39 +0800
Subject: [PATCH 1/4] lit visibility changes

---
 third_party/lit/v3_0/BUILD.gn | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index 72a9a715036f5..8a31bcbbecb80 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//radium/browser/ui/webui/webui.gni")
 import("//tools/typescript/ts_library.gni")
 import("//ui/webui/resources/tools/bundle_js.gni")
 import("//ui/webui/resources/tools/generate_code_cache.gni")
@@ -131,6 +132,8 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
   ]
+  visibility += radium_lit_visibility
+
   tsconfig_base = "tsconfig_base.json"
   composite = true
   in_files = [
-- 
2.34.1

