From 4347c9284e12636dc30dfc0a9aa31ba9f610b4c0 Mon Sep 17 00:00:00 2001
From: weidongliu <weidongliu@chromium.org>
Date: Fri, 8 Nov 2024 21:46:17 +0800
Subject: [PATCH 2/4] Basic gn file modification

---
 .gn | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gn b/.gn
index 3f65718281973..87767e81a5cba 100644
--- a/.gn
+++ b/.gn
@@ -3,6 +3,7 @@
 # file, run "gn help dotfile" at the command line.
 
 import("//build/dotfile_settings.gni")
+import("//radium/dotfile_settings.gni")
 import("//third_party/angle/dotfile_settings.gni")
 
 # The location of the build configuration file.
@@ -173,3 +174,5 @@ exec_script_allowlist =
       "//tools/grit/grit_rule.gni",
       "//tools/gritsettings/BUILD.gn",
     ]
+
+exec_script_allowlist += radium_dotfile_settings.exec_script_allowlist
-- 
2.34.1

