From 8642376a377285acd67f6b3929bb7b6ed44acce2 Mon Sep 17 00:00:00 2001
From: weidongliu <weidongliu@chromium.org>
Date: Fri, 8 Nov 2024 21:46:36 +0800
Subject: [PATCH 4/4] lit visibility changes

---
 third_party/lit/v3_0/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index 2380b3f217230..d7eb8640d5899 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//radium/browser/ui/webui/webui.gni")
 import("//tools/typescript/ts_library.gni")
 import("//ui/webui/resources/tools/bundle_js.gni")
 import("//ui/webui/resources/tools/generate_grd.gni")
@@ -99,6 +100,9 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
   ]
+
+  visibility += radium_lit_visibility
+
   composite = true
   in_files = [
     "lit.ts",
-- 
2.34.1

